src/V3EmitCBase.h

```diff
@@ -71,7 +71,7 @@ class EmitCBaseVisitor VL_NOT_FINAL : public AstNVisitor {
         return v3Global.opt.prefix();
     }
 
-    static bool isConstPoolMod(AstNode* modp) {
+    static bool isConstPoolMod(const AstNode* modp) {
```

Comment:
I suggest push this in a pre-patch as not directly related (I think).

Reply:
I need to call it in this patch with a const pointer, so kind of related, but not particularly interesting indeed.